### PR TITLE
[i18n-IT] Aligned translation of new features

### PR DESCRIPTION
**Pull request by hrumag**
Good morning.
@erciccione, @serhack, please review.

### CODE REVIEW

File: app/src/main/res/values-it/help.xml [hunk 1]
```diff
@@ -159,24 +159,17 @@
...
     ]]></string>
 
     <string name="help_wallet"><![CDATA[
         <h1>Il portafoglio</h1>
-        <h2>Street Mode</h2>
-        <p>Street mode can be enabled/disabled in the menu or Gunther\'s head icon. In this mode, your
-        balance is not shown on any screen so you can safely use your wallet on the street, a pub or
-        other public place. Previous transactions are also hidden. New transactions will be shown, so
-        you can see that you have sent/received sweet Moneroj!</p>
+        <h2>Modalità strada</h2>
```
Comment (serhack) [inline app/src/main/res/values-it/help.xml:221]: Funny.. :- ) "strada"

**Review by serhack — APPROVED** [review]

**Review by erciccione — COMMENTED** [review]
Could you keep the same formatting of the original strings? for consistency and readability (instead of one line for paragraph)

**Comment by hrumag** [comment]
It's pretty difficult since I'm using vim and the formatting (CR) depends on the width of the window.
Let me know if there are problems.

**Comment by erciccione** [comment]
Now looks like the build fails

**Comment by hrumag** [comment]
All should work now.
I took the occasion to squash.

File: app/src/main/res/values-it/strings.xml [hunk 2]
```diff
@@ -357,6 +357,6 @@
     <string name="status_wallet_connect_wrongversion">Versione del nodo incompatibile - si prega di aggiornare!</string>
 
     <string name="menu_info">Dettagli</string><!--Changed to: Show Secrets!-->
```
Comment (m2049r) [inline app/src/main/res/values-it/strings.xml]: could please (re-)translate this: it was changed from "Details" to "Show Secrets!"

**Review by m2049r — CHANGES_REQUESTED** [review]

**Review by m2049r — APPROVED** [review]

**Pull request merged**